deps/minizip/zip.c

```diff
@@ -526,8 +526,8 @@ local ZPOS64_T zip64local_SearchCentralDir(const zlib_filefunc64_32_def* pzlib_f
         break;
       }
 
-      if (uPosFound!=0)
-        break;
+    if (uPosFound!=0)
+      break;
   }
```

Comment:
Actually this gives a warning because there is a for statement without curly braces on top of it. So the compiler warns to show that this doesn't belong to the for. Maybe it's eve nbetter to guard the for with {}. Which one do you prefer?